hibernate-core/src/main/java/org/hibernate/query/internal/FetchMementoJpa.java

```diff
@@ -19,10 +20,12 @@
 
 	private final NavigablePath fetchPath;
 	private final String attributePath;
+	private final List<String> columnNames;
 
-	public FetchMementoJpa(NavigablePath fetchPath, String attributePath) {
+	public FetchMementoJpa(NavigablePath fetchPath, String attributePath, List<String> columnNames) {
```

Comment:
Here again, the only use of the passed `columns` is to pass them to `FetchBuilderJpa` which seems to never use them.